scripts/database/create_tables.sql

```diff
@@ -237,3 +237,25 @@ BEGIN
       IS_ACTIVE                       VARCHAR(50) NULL,
     );
 END
+
+-- Table: SCREENING_LKP
+
+IF NOT EXISTS
+(
+    SELECT *
+    FROM INFORMATION_SCHEMA.TABLES
+    WHERE TABLE_SCHEMA = 'dbo'
+    AND TABLE_NAME = 'SCREENING_LKP'
+)
+BEGIN
+    CREATE TABLE SCREENING_LKP
+    (
+      SCREENING_ID                      BIGINT NOT NULL,
+        CONSTRAINT PK_SCREENING_ID
+          PRIMARY KEY (SCREENING_ID),
+      SCREENING_NAME                    VARCHAR(100) NOT NULL,
+      SCREENING_TYPE                    VARCHAR(100) NULL,
+      SCREENING_ACRONYM                 VARCHAR(100) NULL,
+      SCREENING_WORKFLOW_ID             VARCHAR(100) NULL,
+    );
+END
```

Comment:
Can we add an insert statement to populate this table with the expected screening